### PR TITLE
Adds an entry announcing the Amethyst Foundation and forums

### PR DESCRIPTION
**Pull request by fhaynes**

### CODE REVIEW

File: src/content/blog/non-profit-announce.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Hello and welcome to a special edition of This Week in Amethyst! We have some exciting news to share with the community.
+
+## Amethyst Foundation
+
+We are happy to announce the formation of the Amethyst Foundation. It is a non-profit dedicated to supporting the Amethyst engine and its associated technologies.
+
+As Amethyst grows and evolves, we will continue to depend on the contributions of volunteers. Until now, we haven’t had a clear way for people to contribute other than by coding; now we do! You can donate directly to the Amethyst Foundation via the form below. We thank you for any contribution you can make!
+
+<script src="https://donorbox.org/widget.js" paypalExpress="true"></script><iframe src="https://donorbox.org/embed/amethyst-founding?amount=10" height="685px" width="100%" style="max-width:400px; min-width:310px; max-height:none!important" seamless="seamless" name="donorbox" frameborder="0" scrolling="no" allowpaymentrequest></iframe>
```
Comment (LucioFranco) [inline src/content/blog/non-profit-announce.md:19]: Just want to verify that you tested this?

**Review by LucioFranco — APPROVED** [review]
👍

File: src/content/blog/non-profit-announce.md (hunk 1 above)
Comment (dotellie) [reply src/content/blog/non-profit-announce.md:19]: It's tested.

**Review by dotellie — APPROVED** [review]
bors r+

**Comment by dotellie** [comment]
bors r+

... again

**Comment by LucioFranco** [comment]
🔒 Permission denied

Existing reviewers: click here to make magnonellie a reviewer

File: src/content/blog/non-profit-announce.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
++++
+title = "Amethyst Foundation Created"
+description = "Describes the Amethyst Foundation and new forums"
+date = 2018-11-23
+aliases = ["/2018/11/23/non-profit-announce.html"]
```
Comment (Moxinilian) [inline src/content/blog/non-profit-announce.md:5]: Why?

File: src/content/blog/non-profit-announce.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
++++
+title = "Amethyst Foundation Created"
+description = "Describes the Amethyst Foundation and new forums"
```
Comment (Moxinilian) [inline src/content/blog/non-profit-announce.md:3]: "We are proud to introduce the Amethyst Foundation and our new development forums!"

**Review by Moxinilian — COMMENTED** [review]
I would have liked to be able to review before a merge...

**Pull request merged**